### ISSUE TITLE
[feature request] add a "has" operator for strings

### ISSUE THREAD

**Issue by atomoton** [nushell/nushell#458]
Would love to be able to do something like this: 

`ls | where name has "final" | sort-by name`

to be able to list only files where the name string contains the pattern.

**Comment by Akito13** [comment]
@atomoton As an alias to "==" or something more advanced?

**Comment by Akito13** [comment]
Sounds like a good idea. 

How about:

```
ls | where name has "dong" and startswith "ding" and has not "bummer" and endswith "bang" | sort-by name
```

**Comment by oskarskog** [comment]
Wouldn't the proper way of doing this be to allow passing Nu-script predicates to the verbs, like in rust e.g. ```names.iter().filter(|n| n.starts_with("Harry")).collect()```. Don't know how the syntax would look like since Nu-script doesn't exist yet, but this seems to be the functional way of doing this. 

Nu script -> #291

**Comment by Akito13** [comment]
I'm all for expressiveness.

**Comment by obust** [comment]
It seems like the nu blocks which are [designed as closures](https://github.com/nushell/nushell/issues/291#issuecomment-524674045) could be used as follows:
`where { has $it.name "dong" && startswith $it.name "ding" && endswith $it.name "bang" }`

**Comment by wycats** [comment]
@androbtech yep, I'm a fan of something like this. I don't mind regex, but I think we can do better for simple cases.

**Comment by altmind** [comment]
wouldn't "contains" be a better name?

**Comment by bndbsh** [comment]
@atomoton this should be available now since #997

**Comment by atomoton** [comment]
Fixed in #997
